readthedocs/projects/tasks.py

```diff
@@ -1353,19 +1366,20 @@ def _manage_imported_files(version, path, commit):
 
 
 @app.task(queue='web')
-def send_notifications(version_pk, build_pk):
+def send_notifications(version_pk, build_pk, send_email=True):
```

Comment:
This should probably default to `False`, so that we don't accidentally send users email. Then we can just update the places where we _do_ want to send email.